src/app/organizations/registerOrganization/register-organization.component.html

```diff
@@ -21,15 +21,17 @@ <h1 mat-dialog-title>{{ title }}</h1>
   </p>
   <app-alert></app-alert>
   <form [formGroup]="registerOrganizationForm">
-    <mat-form-field>
+    <mat-form-field [matTooltip]="data.organization?.status === Organization.StatusEnum.APPROVED ? 'Contact Dockstore to change name' : ''">
       <input type="text" matInput formControlName="name" placeholder="Name" required />
```

Comment:
Isn't this still going to allow editing in the UI. Even if it will eventually fail in the API?

Reply:
We talked about this on slack 😢 Reactive forms are weird, had to disable this in the component.ts

Reply:
We did, but I didn't realize the fix was to change it in the component; guess I skimmed over it.